templates/monitoring/kube_state_metrics_rhsso_alerts.yaml

```diff
@@ -13,7 +13,7 @@ spec:
             message: The {{ "{{" }} $labels.container {{ "}}" }} pod has been using {{ "{{" }} printf "%.0f" $value {{ "}}" }}% of available memory for longer than 15 minutes.
             scaling_plan: https://github.com/integr8ly/middleware-load-testing/blob/master/sops/sso-scaling.md
           expr: |
-            sum by(container) (label_replace(container_memory_usage_bytes{container_name!="",namespace="{{ index .Params "Namespace" }}"}, "container", "$1", "container_name", "(.*)")) / sum by(container) (kube_pod_container_resource_limits_memory_bytes{namespace="{{ index .Params "Namespace" }}"}) * 100 > 90
+            sum by(container) (container_memory_usage_bytes{container!="",namespace="{{ index .Params "Namespace" }}"}) / sum by(container) (kube_pod_container_resource_limits_memory_bytes{namespace="{{ index .Params "Namespace" }}"}) * 100 > 90
```

Comment:
@austinmartinh were you able to successfully trigger this alert for RHSSO? It looks like there aren't any resource requests or limits on the Keycloak pods/containers.

@david-martin FYI. If I'm right, should there be resource limits, or should we remove the alert?

Reply:
SSO used to have resource limits when installed with the older integreaty/keycloak-operator.
So this alert should be removed as it's of no use in it's current state.
Also, it's already been marked for removal for a different reason (downstream issue https://issues.redhat.com/browse/INTLY-5491), so @austinmartinh let's remove it.